.rubocop_gradual.lock

```diff
@@ -0,0 +1,132 @@
+{
+  "Gemfile:3311641552": [
+    [19, 3, 4, "Security/Eval: The use of `eval` is a serious security risk.", 2087429787],
+    [21, 3, 4, "Security/Eval: The use of `eval` is a serious security risk.", 2087429787],
+    [23, 3, 4, "Security/Eval: The use of `eval` is a serious security risk.", 2087429787]
```

Comment:
I'm sure you are familiar with lockfiles, and how they are useful.  Using a lockfile for RuboCop violations allows for gradually fixing violations in a way that the rubocop TODO pattern does not.

It's almost a perfect tool, and every project should be using it, IMO:
https://github.com/skryukov/rubocop-gradual